src/layouts/sidebar-sidecar/components/table-of-contents/use-active-section.ts

```diff
@@ -2,24 +2,7 @@ import { useEffect, useRef, useState } from 'react'
 import { TableOfContentsHeading } from './types'
 import { useCurrentProduct } from 'contexts'
 import useCurrentPath from 'hooks/use-current-path'
-import getCSSVariableFromDocument from 'lib/get-css-variable-from-document'
-
-const NAVIGATION_HEADER_HEIGHT_FALLBACK = 68
-
-/**
- * The sticky header has a specific height and we care about headings that are
- * visible below it. This function calculates the height of the header based on
- * a CSS variable.
- *
- * TODO: this may need to be refactored when we address the brittleness of our
- * header height.
- */
-const getFullNavHeaderHeight = () => {
-	return getCSSVariableFromDocument('--navigation-header-height', {
-		asNumber: true,
-		fallback: NAVIGATION_HEADER_HEIGHT_FALLBACK,
-	}) as number
-}
+import getFullNavHeaderHeight from 'lib/get-full-nav-header-height'
```

Comment:
Pulled this out to `lib` as we have a second use for it, as part of using `use-scroll`.